scripts/debian-packaging.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+#
+# This script can be used to invoke OpenZFS build from native Debian
+# packaging.
```

Comment:
I think it would be desirable to switch the `make deb` target to using the native packaging.  This way it's be regularly exercised by the CI and as part of the PR process.  Since we'll probably need to make some small CI tweaks to handle this I'm fine leaving it for a followup PR.

Reply:
Sure, I'll create a follow up PR for this.